cluster/src/main/java/org/apache/iotdb/cluster/log/manage/RaftLogManager.java

```diff
@@ -100,6 +100,16 @@ public RaftLogManager(StableEntryManager stableEntryManager, LogApplier applier,
         .scheduleAtFixedRate(this::checkDeleteLog, logDeleteCheckIntervalSecond,
             logDeleteCheckIntervalSecond,
             TimeUnit.SECONDS);
+
+    /**
+     * flush log to file periodically
+     */
+    int logFlushTimeIntervalMS = ClusterDescriptor.getInstance().getConfig()
+        .getForceRaftLogPeriodInMS();
+    if (ClusterDescriptor.getInstance().getConfig().isEnableRaftLogPersistence()) {
+      executorService.scheduleAtFixedRate(this::flushLogPeriodically, logFlushTimeIntervalMS,
+          logFlushTimeIntervalMS, TimeUnit.MILLISECONDS);
```

Comment:
shutdown this when the manager is closed.

Reply:
Thanks for your advice. I will use a new executor service for raft log writing.